### PR TITLE
[Python] Support for CherryPy Collection Framework 

### PR DESCRIPTION
**Pull request by karan-batavia**

### CODE REVIEW

File: src/main/scala/ai/privado/languageEngine/python/tagger/collection/CherryPyTagger.scala [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+    "",
+    CatLevelOne.COLLECTIONS,
+    "",
+    Language.PYTHON,
+    Array()
+  )
+
+  override def run(builder: DiffGraphBuilder): Unit = {
+    val methodToUrlCache = cpg.call
+      .methodFullName("cherrypy.py:<module>.dispatch.RoutesDispatcher.connect")
```
Comment (khemrajrathore) [inline src/main/scala/ai/privado/languageEngine/python/tagger/collection/CherryPyTagger.scala:38]: Is this always fixed?

Comment (karan-batavia) [reply src/main/scala/ai/privado/languageEngine/python/tagger/collection/CherryPyTagger.scala:38]: Yes

File: src/main/scala/ai/privado/languageEngine/python/tagger/collection/CherryPyTagger.scala [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+        .headOption
+        .getOrElse("")
+        .replaceAll("(\"|')", "")
+    val action =
+      call.argument.where(_.argumentName("action")).isLiteral.code.l.headOption.getOrElse("").replaceAll("(\"|')", "")
+    (path, action)
+  }
+
+  private def getActionMethod(action: String): Option[Method] = {
+    cpg.method.nameExact(action).headOption
```
Comment (khemrajrathore) [inline src/main/scala/ai/privado/languageEngine/python/tagger/collection/CherryPyTagger.scala:89]: This looks very broad

Comment (khemrajrathore) [reply src/main/scala/ai/privado/languageEngine/python/tagger/collection/CherryPyTagger.scala:89]: You can first check for method in same file, if not then do a global search

**Review by khemrajrathore — APPROVED** [review]
LGTM

**Review by khemrajrathore — APPROVED** [review]
LGTM

**Pull request merged**